data/simple-icons.json

```
@@ -38,6 +38,13 @@
 		"hex": "40AEF0",
 		"source": "https://www.1001tracklists.com"
 	},
+	{
+		"url": "https://115.com/",
+		"title": "115生活",
+		"slug": "115",
+		"hex": "FFC808",
+		"source": "https://w.115.com/m_r/web/static/images/icon_logo_cloud.svg?_vh=abd6a19_2"
```

```suggestion
		"source": "https://w.115.com/m_r/web/static/images/icon_logo_cloud.svg"
```

Remove unnecessary search param.

---

It's fixed now.